crates/vfio-ioctls/src/vfio_device.rs

```diff
@@ -1190,3 +1088,4 @@ impl Drop for VfioDevice {
         self.container.put_group(self.group.clone());
     }
 }
+//>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

Comment:
You introduced something that might come from a git rebase.

Reply:
fixed